### DISCUSSION TITLE
Tasmota Devices not working on MQTT with Home Assistant

### DISCUSSION THREAD

**Discussion by stefan-west1** [arendst/Tasmota#14840] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Hi all. Bit of an odd one. Got a load of light switches running tasmota. Been connected to Home Assistant via mqtt and been working perfectly for over a year now. Woke up this morning with all the lights greyed out. Tried restarting home assistant, restarting mosquitto service, updating everything and even deleting and re adding the mqtt integration. Still no luck. The light switches themselves are succesfully connected to the mosquitto service, as they flash if they're not. I turned the power off to all the switches and back on again and this has bought back the status via mqtt but not the binary switch. Nothing at all has changed in regards to the tasmotaconfiguration. Any ideas whats going on?

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): Generic Wifi Light Switch
- [ ] Tasmota binary firmware version number used: 8.3.1 - 11.0.0.1 (Updated one light switch to be sure that wasn't the issue)
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: 
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
00:17:46.430 CMD: Backlog Template; Module; GPIO 255
00:17:46.491 MQT: stat/SW_MBW/C/RESULT = {"NAME":"Innens 1 Gang 1 Way","GPIO":[544,1,1,32,224,1,1,1,1,1,288,1,1,1],"FLAG":0,"BASE":18}
00:17:46.701 MQT: stat/SW_MBW/C/RESULT = {"Module":{"0":"Innens 1 Gang 1 Way"}}
00:17:46.958 MQT: stat/SW_MBW/C/RESULT = {"GPIO0":{"544":"LedLink"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"32":"Button1"},"GPIO4":{"224":"Relay1"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"0":"None"},"GPIO13":{"0":"None"},"GPIO14":{"288":"Led1"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
00:18:06.819 MQT: stat/SW_MBW/C/STATUS0 = {"Status":{"Module":0,"DeviceName":"Master Bedroom Ensuite","FriendlyName":["En-suite Light Switch"],"Topic":"SW_MBW/C","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0},"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-lite.bin","RestartReason":"Software/System restart","Uptime":"0T00:04:46","StartupUTC":"2022-02-13T23:13:20","Sleep":50,"CfgHolder":4617,"BootCount":135,"BCResetTime":"2020-05-26T10:32:45","SaveCount":5721,"SaveAddress":"F7000"},"StatusFWR":{"Version":"11.0.0.1(lite)","BuildDateTime":"2022-02-13T22:00:30","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"456/699"},"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["It burns when IP",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["4002A009","2805C80001000600003C5A00000000000000","00000280","00006000","00004000"]},"StatusMEM":{"ProgramSize":503,"Free":500,"Heap":29,"ProgramFlashSize":1024,"FlashSize":2048,"FlashChipId":"1540C8","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8F000783","04108001","00000006","010001C0","00000000","00004004","00000000","00000020"],"Drivers":"1,2,3,4,9,10,12,16,20,21,37","Sensors":"3"},"StatusNET":{"Hostname":"SW-MBW-C-2236","IPAddress":"192.168.86.104","Gateway":"192.168.86.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.86.1","DNSServer2":"0.0.0.0","Mac":"D8:BF:C0:52:C8:BC","Webserver":2,"HTTP_API":1,"WifiConfig":2,"WifiPower":17.0},"StatusMQT":{"MqttHost":"192.168.86.49","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_52C8BC","MqttUser":"hassio","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4},"StatusTIM":{"UTC":"2022-02-13T23:18:06","Local":"2022-02-14T00:18:06","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"08:00","Sunset":"18:08"},"StatusSNS":{"Time":"2022-02-14T00:18:06"},"StatusSTS":{"Time":"2022-02-14T00:18:06","Uptime":"0T00:04:46","UptimeSec":286,"Vcc":3.499,"Heap":28,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"It burns when IP","BSSId":"3C:28:6D:55:C8:5E","Channel":11,"Mode":"11n","RSSI":54,"Signal":-73,"LinkCount":1,"Downtime":"0T00:00:05"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
I have no idea. Happened randomly

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Turn the lights on and off via mqtt

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by ascillato2** [comment, marked as answer]
Since you haven't had any reply here, have you tried addressing this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4)?
